homeassistant/components/fritz/common.py

```diff
@@ -413,6 +417,14 @@ async def async_trigger_reconnect(self) -> None:
         """Trigger device reconnect."""
         await self.hass.async_add_executor_job(self.connection.reconnect)
 
+    async def async_trigger_set_guest_password(
+        self, password: str | None, length: int | None
```

Comment:
```suggestion
        self, password: str | None, length: int
```

sorry, did oversee this in the last review 🙄